ui/main.qml

```diff
@@ -214,6 +215,15 @@ ApplicationWindow {
             applicationWindow.settingsLoaded()
         }
     }
+    Connections {
+        target: profileModel
```

Comment:
You could add `enabled: !appSettings.showBackupMessage` here so that this Connections is only active when that flag is up, removing a tiny bit of usage.
You could obviously remove the `if` below after